src/cluster/predefine.cpp

```diff
@@ -54,7 +54,7 @@ struct dynamic_converter<predefine_cfg_t> {
     convert(const dynamic_t& source) {
         result_type result;
 
-        const dynamic_t& nodes = source.as_object().at("nodes", dynamic_t::object_t());
+        const dynamic_t& nodes = source.as_object().at("nodes", dynamic_t::empty_object);
```

Comment:
Don't you think it's a bad design of 'at' method of dynamic?
It should distinguish between rvalue and lvalue. And it should try to return by value in case of rvalue.